utils.js

```diff
@@ -1,11 +1,147 @@
 'use strict';
 
-var utils = {
+/**
+ * Uses abstract comparison (==) NOT strict comparison (===)
+ */
+function contains(list, item) {
```

Comment:
@odino I want to discuss this with you and @shidhincr.

I wanted to avoid lodash because the dependencies problem in the browser: if I assume it is there (and it isn't), the code will not throw an error until I reach this point.

Do I have a way to "force" a package to be loaded? Ho do you do it in the browser/with bower?